ci-operator/config/jboss-fuse/camel-k-test-container/jboss-fuse-camel-k-test-container-main__camel-k-ocp4.14-lp-rosa.yaml

```diff
@@ -33,22 +33,24 @@ tests:
     env:
       FIREWATCH_CONFIG: |
         [
-          {"step": "aws-*", "failure_type": "all", "classification": "Infrastructure AWS", "jira_project": "LPTOCPCI", "group": {"name": "cluster", "priority": 1}},
-          {"step": "rosa-*", "failure_type": "all", "classification": "Infrastructure Provisioning - Cluster", "jira_project": "LPTOCPCI", "group": {"name": "cluster", "priority": 1}},
-          {"step": "install-operators", "failure_type": "all", "classification": "Infrastructure Provisioning - Operators", "jira_project": "LPTOCPCI", "group": {"name": "operator", "priority": 1}},
-          {"step": "gather-*", "failure_type": "pod_failure", "classification": "Other", "jira_project": "LPTOCPCI", "group": {"name": "cluster", "priority": 2}},
-          {"step": "gather-*", "failure_type": "test_failure", "classification": "NONE", "jira_project": "NONE", "ignore": "true"},
-          {"step": "openshift-*", "failure_type": "all", "classification": "Other", "jira_project": "LPTOCPCI", "group": {"name": "cluster", "priority": 2}},
-          {"step": "camel-k-interop-tests", "failure_type": "test_failure", "classification": "Test Failure", "jira_project": "CMLK", "group": {"name": "operator", "priority": 2}}
+          {"step": "aws-*", "failure_type": "all", "classification": "Infrastructure AWS", "jira_project": "LPTOCPCI", "group": {"name": "cluster", "priority": 1}, "jira_additional_labels": ["4.14-lp","rosa-lp","camel-k-lp"]},
+          {"step": "rosa-*", "failure_type": "all", "classification": "Infrastructure Provisioning - Cluster", "jira_project": "LPTOCPCI", "group": {"name": "cluster", "priority": 1}, "jira_additional_labels": ["4.14-lp","rosa-lp","camel-k-lp"]},
```

Comment:
rosa-cluster-deprovision/
rosa-cluster-provision/
rosa-cluster-wait-ready-nodes/
rosa-cluster-wait-ready-operators
rosa-conf-idp-htpasswd
rosa-sts-account-roles-create/
rosa-sts-account-roles-delete/

here is the list of steps that run starting with rosa*, I think we need to break up this section into more specific rules, I don't believe all of these should be considered priority 1.
- I believe only the steps starting with rosa-cluster should be priority 1 and the others should be priority 2

For example if rosa-cluster-provision fails rosa-cluster-deprovision will also fail. If they are both priority 1 then we will report two tickets, 1 ticket correctly capture the provision failure and 1 ticket incorrectly capturing the deprovision failure.

I'm thinking steps starting with rosa might look like this

rosa-cluster-provision = priority 1
rosa-cluster-deprovision = priority 2
rosa-cluster-wait* = priority 2
rosa-sts-account-roles-create = priority 3
rosa-sts-account-roles-delete = priority 4
rosa-conf-idp-htpasswd - priority 4

This makes the config longer which I don't love though I can't see a way to prevent incorrectly creating tickets due to cascading failures without doing this.